app.py

```diff
@@ -162,9 +162,15 @@ def static_grid():
 @app.route('/data/thredds/velocity/step/<int:time_step>')
 def thredds_velocity_frame(time_step):
     """ Return the velocity frame corresponding to `time_step`. """
-    vs = tc.fs.velocity_frame(time_step)
-    vs = jsonify_dict_of_array(vs)
-    return json.dumps(vs)
+    try:
+        vs = tc.fs.velocity_frame(time_step)
+        vs = jsonify_dict_of_array(vs)
+        return json.dumps(vs)
+    except Exception as e:
```

Comment:
Yes, that's probably right. We may still want to catch other errors to log the error message even if not in debug mode. Not sure that's necessary, though.